server/controller/root.py

```diff
@@ -2,27 +2,28 @@
 Root Controller
 """
 import functools
-import controller
 
 from bottle import abort, post, redirect, request, response, route, static_file
 from kajiki_view import kajiki_view
 from markupsafe import Markup
 
+from helpers.config_manager import ConfigManager
```

Comment:
Can you do something like: from helpers.config_manager.ConfigManager import user_management_interface as user_interface or something? The only uses of ConfigManager are configManager.user_management_interface